pretty-fast.js

```diff
@@ -845,6 +845,12 @@
       }
 
       ttk = token.type.keyword;
+
+      if (ttk && lastToken && lastToken.type.label == ".") {
+        token.type = acorn.tokTypes.name;
+        ttk = token.type.keyword;
```

Comment:
why is this line needed?

Reply:
just removed it because it is likely redundant

Reply:
I believe it actually is needed because `token.type` is reassigned, so `token.type.keyword` has changed and thus `ttk` needs to be updated.

Reply:
hmm, i see what you're saying. Could you file a new PR with that change and a test that demonstrates it.

Reply:
`ttk` is only used once on line 867, which will never be reached in the case of reassigning `token.type` because `belongsOnStack` doesn't return true for the `name` token type. So currently there is no code path that is bugged and there's no test I could write for it. However, it's still an issue because in the future someone might use `ttk` and a bug will be introduced because it has an incorrect value in some cases. Should I just submit the PR without a test?

Reply:
sure